.grabl/automation.yml

```diff
@@ -60,9 +60,10 @@ build:
         export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
         export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
         bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
-        bazel test //test/behaviour/connection/... --test_output=errors
+        bazel test //test/behaviour/connection/... --test_output=errors --jobs=1
```

Comment:
Because `SessionTest` and `DatabaseTest` each create 12 databases, running the two in parallel can cause the two Grakn processes combined to use over 2.1GB of memory. This is more than Grabl has.

We work around this limitation by executing the tests serially using `--jobs=1`.